client/lib/two-step-authorization/index.js

```diff
@@ -35,6 +35,7 @@ function TwoStepAuthorization() {
 	this.smsResendThrottled = false;
 	this.bumpMCStat = function( eventAction ) {
 		analytics.mc.bumpStat( '2fa', eventAction );
+		analytics.tracks.recordEvent( 'calypso_two_step', { eventAction } );
```

Comment:
This event with property `eventAction` will be rejected by tracks 😬 

> Property names should also be lowercase and use underscores instead of dashes

Reply:
Thank you for this! 👍